docs/DeveloperGuide.md

```diff
@@ -640,84 +639,40 @@ Guarantees: Viewing of application is successful
+#### Use case: UC15 - Delete a user profile item 
 
-Precondition: User already has an existing list of user profile items <br/>
-Guarantees: Deletion of user profile item is successful
+* Similar to deleting a company except user is deleting a user profile item.
 
-**MSS**
+#### Use case: UC16 - Edit a user profile item
 
-1.  User requests to delete a user profile item from the user profile.
-2.  InternHunter removes the user profile item from the user profile. <br/>
-    Use case ends.
+* Similar to editing a company except user is editing a user profile item.
 
-**Extensions**
-
- 1a. InternHunter detects an error in the input format. <br/>
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
-  Use case resumes from step 1.
-
- 1b. InternHunter detects an invalid index. <br/>
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
-  Use case resumes from step 1.
+#### Use case: UC17 - View a user profile item
 
-#### Use case: UC14 - Edit a user profile item
+* Similar to viewing a company except  user is viewing a user profile item.
 
-Precondition: User already has an existing list of user profile items <br/>
-Guarantees: Editing of user profile item is successful
+#### Use case: UC17 - Finding user profile items
 
-**MSS**
+* Similar to finding companies except user is finding user profile items.
 
-1.  User requests to edit the details of a user profile item and provides details.
-2.  InternHunter edits the user profile item in the user profile. <br/>
-    Use case ends.
+#### Use case: UC19 - List all user profile items
 
-**Extensions**
+* Similar to listing all companies except user is listing all user profiles items
 
- 1a. InternHunter detects an error in the input format. <br/>
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
-  Use case resumes from step 1.
 
- 1b. InternHunter detects an invalid index. <br/>
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
-  Use case resumes from step 1.
-
-#### Use case: UC15 - View a user profile item
-
-Precondition: User already has an existing list of user profile items <br/>
-Guarantees: Viewing of user profile item is successful
-
-**MSS**
-
-1.  User requests to view the details of a user profile item.
-2.  InternHunter shows the details of this user profile item. <br/>
-    Use case ends.
-    
-**Extensions**
-
- 1a. InternHunter detects an error in the input format. <br/>
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
-  Use case resumes from step 1.
-
- 1b. InternHunter detects an invalid index. <br/>
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
-  Use case resumes from step 1.
+#### Use case: UC15 - Matching skills in user profile to internship requirements
```

Comment:
Remaining numbers seem to not be updated?

Reply:
yes this is deliberate, I stopped updating the index because I want to know the current refactoring is appropriate